components/contribution-flow/StepDetails.js

```diff
@@ -248,6 +295,8 @@ StepDetails.propTypes = {
   onChange: PropTypes.func,
   /** If true, the select for interval will be disabled */
   disabledInterval: PropTypes.bool,
+  tierName: PropTypes.string,
+  collectiveSlug: PropTypes.string,
```

Comment:
Because not passing it would crash the redirect

```suggestion
  collectiveSlug: PropTypes.string.isRequired,
```